tests/unit/matrix/test_assembler.py

```diff
@@ -1574,6 +1579,34 @@ def it_provides_the_order(self, request):
     def it_provides_the_element_values_to_help(self, request, _marginal_prop_):
         assert _SortRowsByMarginalHelper(None, None)._element_values == "a"
 
+    @pytest.mark.parametrize(
+        "marginal, marginal_prop_name",
+        (
+            (MARGINAL.SCALE_MEAN, "rows_scale_mean"),
+            (MARGINAL.SCALE_MEAN_STDDEV, "rows_scale_mean_stddev"),
+            (MARGINAL.SCALE_MEDIAN, "rows_scale_median"),
+        ),
+    )
+    def it_provides_the_marginal_to_help(self, request, marginal, marginal_prop_name):
+        second_order_measures_ = instance_mock(request, SecondOrderMeasures)
+        setattr(second_order_measures_, marginal_prop_name, "foo")
+        _order_spec_ = instance_mock(request, _OrderSpec, marginal=marginal)
+        dimensions_ = [instance_mock(request, Dimension, order_spec=_order_spec_)]
+        order_helper = _SortRowsByMarginalHelper(dimensions_, second_order_measures_)
+
+        assert order_helper._marginal == "foo"
+
+    def but_it_raises_on_unknown_marginal(self, request):
+        second_order_measures_ = instance_mock(request, SecondOrderMeasures)
```

Comment:
If you use the same mock twice (or more) I would make it a fixture. The inlines are only for cases where there's just the one test that's ever going to use it. This particular case is one of the reasons I went from `second_order_measures` to the shorter `measures`. The longer version works okay in the code but parameter-list real estate is more precious in the unit tests it seems :)

It looks like there are at least four mocks here that could profitably be moved to the `fixture components` section below.